examples/fetch/package.json

```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "scripts": {
     "start": "parcel index.html --open",
+    "clean": "echo 'No clean'",
```

Comment:
Can we remove this?

Reply:
It is required for the root clean script to run, otherwise it fails. In turn, the typescript verification script requires the root clean script.

Reply:
Actually, scratch that, the verify script no longer requires this (it did at some point). 

However, if you're OK I think we should still leave this in, because it does fix the root clean script.